x-pack/plugins/ml/public/application/datavisualizer/file_based/components/results_view/results_view.tsx

```diff
@@ -103,7 +90,17 @@ export const ResultsView: FC<Props> = ({
           <EuiSpacer size="m" />
 
           <EuiPanel data-test-subj="mlFileDataVisFileStatsPanel">
-            <EuiTabbedContent tabs={tabs} initialSelectedTab={tabs[0]} onTabClick={() => {}} />
+            <EuiTitle size="s">
+              <h2 data-test-subj="mlFileDataVisStatsTitle">
+                <FormattedMessage
+                  id="xpack.ml.fileDatavisualizer.resultsView.fileStatsName"
+                  defaultMessage="File stats"
+                />
+              </h2>
+            </EuiTitle>
+
+            <FieldsStatsGrid results={results} />
+            {/* <EuiTabbedContent tabs={tabs} initialSelectedTab={tabs[0]} onTabClick={() => {}} />*/}
```

Comment:
Is this commented out content needed?

Reply:
Thanks for catching that. Removed here https://github.com/elastic/kibana/pull/87598/commits/bc29f197bd057f642315caaa2180a615712b9670#diff-5650b96d55c82f01621ecc4766b2e04e67fbc2083c567bb57cac92a8518eac68L103.